src/aerialmap_display.cpp

```diff
@@ -411,18 +412,30 @@ void AerialMapDisplay::assembleScene()
       // tile width/ height in meter
       double const tile_w_h_m = getTileWH();
 
-      // determine position of this tile relative to the robot's position
-      // note: We invert the y-axis so that positive y corresponds to north. See transformAerialMap().
-      double const x = (xx - lastTileId_->coord.x) * tile_w_h_m;
-      double const y = -(yy - lastTileId_->coord.y) * tile_w_h_m;
+      // Note: In the following we will do two things:
+      //
+      // * We flip the position's y coordinate.
+      // * We flip the texture's v coordinate.
+      //
+      // For more explanation see the function transformAerialMap()
+
+      // In the AerialMap frame we to have the center tile at the position left-bot = (0,0) and right-top = (1,1)
```

Comment:
we to have?

what's the AerialMap frame?

Reply:
> we to have?

Fixed

> what's the AerialMap frame?

This is defined in transformAerialMap()